dashboard/app/controllers/datablock_storage_controller.rb

```diff
@@ -326,7 +326,11 @@ def use_firebase_storage
   end
 
   private def where_table
-    DatablockStorageTable.where(project_id: @project_id, table_name: params[:table_name])
+    if params[:table_name]
+      DatablockStorageTable.where(project_id: @project_id, table_name: params[:table_name])
+    else
+      raise StudentFacingError, "You must specify a table"
```

Comment:
We should be able to test this in the same test harness thing that we used to test getColumn to make sure that error is being routed all the way to the student console! I will go find after I finish reviewing the others.

Reply:
I mean, we could add a controller test, but I'm pretty 🤷 on that ("tests that don't break aren't good tests"), and we can't do testing that involves crossing the JS <-> backend (we never figured out how to test that).

Reply:
Because when called through the block itself, there's always a table_name sent, yeah? and we just aren't sure how this is getting called without one. is that right?

Reply:
yup, that's IT!